pkg/config/api_config.go

```diff
@@ -69,7 +69,8 @@ type Resource struct {
 
 // Method defines the method of the api
 type Method struct {
-	OnAir              bool `yaml:"onAir"`
+	OnAir              bool     `yaml:"onAir"`
```

Comment:
what's the meaning of onair.......should you add some comment for this variable?

Reply:
done